common/remote/sockfile.cpp

```diff
@@ -2134,6 +2134,7 @@ class CRemoteFile : public CRemoteBase, implements IFile
     {
         flags &= ~(IFSHfull|IFSHread);
         flags |= (unsigned)(shmode&(IFSHfull|IFSHread));
+        flags |= IFSHset;
```

Comment:
Should this be set if setShareMode was called with IFSHnone ?

Reply:
Yes.  IFSHnone is a valid setting.  If setSharedMode() is called then we should honor the value passed in.